### PR TITLE
Resfile Implementation

### PR DESCRIPTION
**Pull request by dtemir**
Add Resfile functionality to the model.

The resfile can limit produced conditional distributions by modifying tensors with logits. It can also initialize sequences with a pre-specified sequence.

Read more in the `seq_des/util` directory

### CODE REVIEW

**Comment by dtemir** [comment]
sorry, wrong repo :)
